pkg/resources/configprovider/schema.go

```diff
@@ -0,0 +1,66 @@
+package configprovider
+
+import (
+	"context"
+	_ "embed"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"go.clever-cloud.com/terraform-provider/pkg"
+	"go.clever-cloud.com/terraform-provider/pkg/attributes"
+)
+
+type ConfigProvider struct {
+	attributes.Addon
```

Comment:
config provider is a bit specific, you can drop attributes.Addon and only use relevant attributes 
(for exemple creation_date and region are not relevant)